minitest/repositores/name_repository.py

```
@@ -0,0 +1,87 @@
+            raise ValidationError('A name with the same value already exists')
+
+        try:
+            # UUID로 사용자 찾기
+            user = self.model.objects.get(id=uuid)
+            # 이름 변경 및 저장
+            user.name = name
+            user.save()
+            print('change name: ', user)
+            return user
+        except self.model.DoesNotExist:
+            raise ValidationError('A user does not exist')
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)